src/main/java/com/criticalsoftware/Announcement.java

```diff
@@ -0,0 +1,68 @@
+package com.criticalsoftware;
+
+import org.bson.types.ObjectId;
+
+import java.time.LocalDateTime;
+
+public class Announcement {
+
+    private String id;
```

Comment:
As the id field is already inherited from the parent class (PanacheMongoEntity), it doesn't need to be explicitly declared in this class.